webform_import.module

```diff
@@ -311,17 +318,22 @@ function _webform_import_import($form, $form_state, $file) {
   $delimiter = $delimiter == "\t" ? "\t" : $delimiter;
   $field_key = $form_state['values']['field_keys'];
 
-  $components_list = array(
-    'UID' => array('form_key' => 'webform_uid'),
-  );
-
-  if ($field_key === 'name') {
-    $components_list['Submission ID'] = array('form_key' => 'webform_sid');
+  // Main portion of fix for #4
```

Comment:
Can you please remove this comment? 

We don't need references to specific issues in the codebase once they are fixed. Only if this was an ongoing issue would be include something like "Remove this until #4 is fixed".

Reply:
I think I was able to edit the PR to make this change. Marking as resolved :)